app.py

```diff
@@ -40,6 +40,8 @@ def exportBag():
     manifest_io = ManifestIO(data, url, workspace, namespace, auth)
     # Check whether the workspace exists in this namespace, otherwise create it.
     r_check_workspace = manifest_io.workspace_exists()
+    if hasattr(r_check_workspace, 'body'):  # is Chalice Response object?
```

Comment:
This is hard to understand. Are you saying if it doesn't time out, there is no body attribute? It seems like a fragile test... I think I would lean towards catching the exception.